datashuttle/datashuttle.py

```diff
@@ -198,6 +161,10 @@ def create_folders(
                 NeuroBlueprint will be created. If "" is passed
                 no datatype will be created.
 
+        top_level_folder :
+                Whether to make the folders in `rawdata` or
+                `derivatives`.
```

Comment:
As a rule, it's better for the order of params in the docstrign to match that in the function signature, so I would move this up as the first param.